test/Tester/ActivationServices/IStorageActivationService.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+using Orleans.Runtime;
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Orleans;
+
+namespace Tester
+{
+    public interface IStorageActivationService<TState>
```

Comment:
I think this name is fine for tests.  I don't think actual activation services will have ActivationService" in their names.  They will likely be more closely related to the behavior they expose, like IPersistentState, IStreamConsumer, or ITransactionalState.